.projenrc.ts

```diff
@@ -97,7 +98,7 @@ const jsiiLibraryProjectOptions: cdk.JsiiProjectOptions = {
     'jsonwebtoken',
     'jwks-rsa',
     'uuid',
-    `aws-cdk-lib@${POINT_SOLUTIONS_CDK_VERSION}`,
+    `aws-cdk-lib@2.195.0`,
```

Comment:
I believe this is typo, you want to use ${POINT_SOLUTIONS_CDK_LIB_VERSION} right?

Reply:
Fixed. Thanks!